### DISCUSSION TITLE
Not able to pull manifests

### DISCUSSION THREAD

**Discussion by Luumonix** [Homebrew/discussions#4289] [Casks] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.0.4
ORIGIN: https://github.com/Homebrew/brew
HEAD: 4d0154fa7926e8a7b06e9b074dd45957e820407c
Last commit: 2 days ago
Core tap origin: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 9984d2778949b441beeb63010366aefd12faa2eb
Core tap last commit: 3 hours ago
Core tap branch: master
Core tap JSON: 01 Mar 13:36 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: octa-core 64-bit arm_firestorm_icestorm
Clang: 14.0.0 build 1400
Git: 2.37.1 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 7.86.0 => /usr/bin/curl
macOS: 13.2.1-arm64
CLT: 14.2.0.0.1.1668646533
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

Is anyone else having an issue trying to pull dependencies to install applications from Brew?

I'm getting:
`==> Fetching dependencies for spoof-mac: mpdecimal, openssl@1.1, readline, sqlite and python@3.11`
`==> Fetching mpdecimal`
`==> Downloading https://ghcr.io/v2/homebrew/core/mpdecimal/manifests/2.5.1`

which hangs, and if I directly try and pull the file:
` {"errors":[{"code":"UNAUTHORIZED","message":"authentication required"}]}`

Any ideas?

### Relevant casks

Have tried: mpdecimal, freetype

**Comment by KikiranLLC** [comment, marked as answer]
Looks like GitHub is having an issue with their packages system.

https://www.githubstatus.com/incidents/bypf7bqdh4g0

**Reply by Luumonix** [reply]
Oh, that's great. Did not think of Github being the issue; I saw other posts on StackOverflow saying it was an issue with network filtering but I had tried on 5 different networks 😂. Thanks!

**Comment by chrishaines** [comment]
It looks like GitHub resolved the issue with packages. Does the packaging job need to be manually kicked off?
